.github/workflows/ci.yml

```diff
@@ -18,13 +18,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+        with:
+          run_install: false
       - uses: actions/setup-node@v4
         with:
           node-version: 18
-          cache: 'yarn'
-      - run: yarn install --frozen-lockfile
-      - run: yarn lint
-      - run: yarn test:ember
+          cache: 'pnpm'
+      - run: pnpm install
```

Comment:
```suggestion
      - run: pnpm install --frozen-lockfile
```

Reply:
this is default in CI https://pnpm.io/cli/install, so the flag isn't needed

Reply:
til 🎉